vars/longevityPipeline.groovy

```diff
@@ -1,5 +1,124 @@
 #!groovy
 
+def runSctTest(Map params){
+    // handle params which can be a json list
+    params = params.params
```

Comment:
if pipelineParams is passed to this function, I can understand this, but this isn't the case

Reply:
now I get it, since a named param is used, this is very confusing...

Reply:
I tried hard, but it looks like there is no other way to make function work properly with first map parameter.